ext/src/Cassandra/Cluster/Builder.c

```diff
@@ -755,6 +968,14 @@ static zend_function_entry cassandra_cluster_builder_methods[] = {
          ZEND_ACC_PUBLIC)
   PHP_ME(ClusterBuilder, withDatacenterAwareRoundRobinLoadBalancingPolicy,
          arginfo_dc_aware, ZEND_ACC_PUBLIC)
+  PHP_ME(ClusterBuilder, withBlackListNodes,
```

Comment:
What do you think about "withBlackListHosts" and "withWhiteListHosts" instead of "Nodes"?